Source/WebInspectorUI/UserInterface/Views/MultiSidebar.js

```diff
@@ -136,6 +137,11 @@ WI.MultiSidebar = class MultiSidebar extends WI.Sidebar
         super.collapsable = allow;
         this.primarySidebar.collapsable = this.collapsable;
     }
+    
+    get actuallyBottomBar()
```

Comment:
`this._actuallyBottomBar` and the getter don't seem to be used anymore. Please remove.